perceval/backends/github.py

```diff
@@ -22,15 +22,16 @@
 #     Alberto Martín <alberto.martin@bitergia.com>
 #
 
+from datetime import datetime
```

Comment:
Please move this line under the last import of packages from the standard library. You should also move the `from time import sleep` line there too.